src/status_im/data_store/realm/transport.cljs

```diff
@@ -4,8 +4,7 @@
 
 (defn get-all
   []
-  (-> (realm/get-all @realm/account-realm :transport)
-      realm/js-object->clj))
+  (realm/all-clj (realm/get-all @realm/account-realm :transport) :transport))
```

Comment:
Do we ever want `get-all` without `all-clj` ?

Reply:
Potentially yes, if one wants for example to select all realm entities of some type (let's say messages) and read one property from each into set or whatever. Makes sense to not force anybody to do the whole object realm->clj conversion just for that.